packages/widgets/src/components/CollapsibleNavBar/NavBarAccordionContent.tsx

```diff
@@ -0,0 +1,34 @@
+import { AccordionContent } from '@ssa-ui-kit/core';
+import { RenderContentProps } from '@ssa-ui-kit/core/dist/components/AccordionGroup/types';
```

Comment:
Do not refer `dist` folder in the source code. This is incorrect. 
If something is missing from the export, add the corresponding export in the package.